src/server.ts

```diff
@@ -1,3 +1,5 @@
+import "instrument"
```

Comment:
do we want to import it all the time or only when `SENTRY_PRIVATE_DSN` is present?

Reply:
All the time. This is how it was before but it was set up incorrectly.